### DISCUSSION TITLE
Is Licence Needed on Kesteral Ngix on Windows host

### DISCUSSION THREAD

**Discussion by tuffboys** [dotnet/aspnetcore#23557] [General]
I am developing app on .net core and want to know about licence needed to host this app on windows with kestrel server and ngix server.

**Comment by davidfowl** [comment]
ASP.NET Core is free to use.

**Reply by tuffboys** [reply]
Yes, it is free but when we are hosting this app on windows 10 system, what about licencing needs?

**Reply by davidfowl** [reply]
That's a windows question right? Not a .NET Core question.

**Reply by tuffboys** [reply]
Right, want to know about Internat cal licence needed or not.

**Reply by davidfowl** [reply]
This isn't the right place to ask that question (I don't even know what the question is really). Are you running on a host or are you hosting your own hardware?
